frontend/src/lib/components/modals/ServerManager.svelte

```diff
@@ -227,11 +245,19 @@
             placeholder="user"
             type="text"
             bind:value={newServerUsername}/>
-          <input
+          {#if maskPassword}
+            <input
+              class="input px-4 h-full"
+              placeholder="pass"
+              type="password"
+            bind:value={newServerPassword}/>
+          {:else}
+            <input
             class="input px-4 h-full"
             placeholder="pass"
             type="text"
             bind:value={newServerPassword}/>
+          {/if}
```

Comment:
This could use a comment explaining that `type` can't be dynamic when using `bind:value`